content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -190,40 +190,40 @@ Rechenschaftspflicht ist wichtig, deshalb wollen wir dieses Jahr mit der Veröff
 
-* Wenn Sie ein Matrix.org-Homeserver-Nutzer sind, können [Sie die Inhalte](https://github.com/element-hq/element-x-ios/pull/587) von Ihrem Client aus melden, und sie werden an unser Sicherheitsteam weitergeleitet. Matrix-Clients wie Element iOS ermöglichen es Nutzern, [Räume zu melden](https://github.com/element-hq/element-ios/pull/7805), und wir wissen, dass dies bald auch in Element X, Element Android und Element Web verfügbar sein wird.  
+* Wenn Sie den Matrix.org-Homeserver nutzen, können [Sie die Inhalte](https://github.com/element-hq/element-x-ios/pull/587) von Ihrem Client aus melden, und sie werden an unser Trust & Safety Team weitergeleitet. Matrix-Clients wie Element iOS ermöglichen es Nutzern, [Räume zu melden](https://github.com/element-hq/element-ios/pull/7805), und wir wissen, dass dies bald auch in Element X, Element Android und Element Web verfügbar sein wird.  
 * Wenn Sie einen anderen Matrix-Server verwenden, können Sie die Meldung ebenfalls von Ihrem Client aus an Ihren Server-Administrator senden.  
-* Wenn Sie kein Matrix-Nutzer sind oder wenn Sie ein Nutzer auf einem anderen Homeserver sind und uns über schädliche Inhalte auf matrix.org informieren möchten, können Sie uns eine E-Mail mit den Informationen, die Sie haben, an abuse@matrix.org schicken. Raum- und Benutzerkennungen sind sehr hilfreich. Bitte schicken Sie uns keine Bildschirmfotos von schädlichen Inhalten \- wir werden Ihnen Bescheid geben, wenn wir weitere Informationen benötigen.
+* Wenn Sie kein Matrix-Nutzer sind oder wenn Sie ein Nutzer auf einem anderen Homeserver sind und uns über schädliche Inhalte auf Matrix.org informieren möchten, können Sie uns eine E-Mail mit den Informationen, die Sie haben, an [abuse@matrix.org](mailto:abuse@matrix.org) schicken. Raum- und Benutzerkennungen sind sehr hilfreich. Bitte schicken Sie uns keine Bildschirmfotos von schädlichen Inhalten \- wir werden Ihnen Bescheid geben, wenn wir weitere Informationen benötigen.
 
-### Als Matrix Server Betreiber
+### Als Matrix-Server-Betreiber
 
-* Die offene Registrierung ist in Synapse [standardmäßig deaktiviert](https://github.com/matrix-org/synapse/commit/58367a9da2539abdbfe4dc817fba5b179b95334b), und wir unterstützen diese Voreinstellung. Wenn Ihr Server eine offene Registrierung anbietet, müssen Sie in ein Sicherheitsteam investieren, um eine angemessene Moderation zu gewährleisten und die Risiken zu minimieren, die entstehen, wenn Sie unbekannten Benutzern die Nutzung Ihres Servers erlauben.  
+* Die offene Registrierung ist in Synapse [standardmäßig deaktiviert](https://github.com/matrix-org/synapse/commit/58367a9da2539abdbfe4dc817fba5b179b95334b), und wir befürworten diese Voreinstellung. Wenn Ihr Server eine offene Registrierung anbietet, **müssen** Sie in ein Trust & Safety Team investieren, um eine angemessene Moderation zu gewährleisten und die Risiken zu minimieren, die entstehen, wenn Sie unbekannten Benutzern die Nutzung Ihres Servers erlauben.  
```

Comment:
- "befürworten", to clarify meaning support in the sense of "endorsing", rather than in the sense of "providing"
- @ara4n I'm not a 100% fan of "müssen" (nor "must" in the original), since it's not a technical requirement nor do I believe global legal requirement, but rather our strong opinion that ethically you really really really should. It might be better phrased as an imperative? "If your server..., invest in T&S to...". wdyt?